opencensus/proto/agent/trace/v1/trace_service.proto

```diff
@@ -30,14 +30,13 @@ option java_outer_classname = "TraceServiceProto";
 option go_package = "github.com/census-instrumentation/opencensus-proto/gen-go/agent/trace/v1";
 
 service TraceService {
+  // After initialization, this RPC must be kept alive for the
+  // entire life of the application. The agent pushes configs
+  // down to applications via a stream of requests.
+  rpc Config(stream ConfigTraceServiceResponse) returns (stream ConfigTraceServiceRequest) {}
```

Comment:
Don't understand this. Is this what we want to do? Request to be named the message that comes from the agent? I think will confuse some people but I think the idea is good. Maybe just comment that requests are coming from the agent, that's why the message sent by the agent is a "request" (or something like that to clarify).

Reply:
Yes that's the idea, I posted https://github.com/census-instrumentation/opencensus-proto/pull/93 that added a note about this.